pkg/activator/handler/context.go

```diff
@@ -48,7 +48,7 @@ func withRevID(ctx context.Context, revID types.NamespacedName) context.Context
 	return context.WithValue(ctx, revIDKey{}, revID)
 }
 
-// revIDFrom retrieves the the revisionID from the context.
-func revIDFrom(ctx context.Context) types.NamespacedName {
+// RevIDFrom retrieves the the revisionID from the context.
+func RevIDFrom(ctx context.Context) types.NamespacedName {
```

Comment:
does this need to be exported? I think all the uses are still in the `pkg/activator/handler/` package so should be ok to keep it as it is?

Reply:
`request_log.go` in `cmd/activator`

Reply:
oh good point, I was mis-reading that! nope this lgtm then - thanks :)